pylint/checkers/base_checker.py

```diff
@@ -109,10 +111,14 @@ def get_full_documentation(self, msgs, options, reports, doc=None, module=None):
         return result
 
     def add_message(
-        self, msgid, line=None, node=None, args=None, confidence=None, col_offset=None
-    ):
-        if not confidence:
-            confidence = UNDEFINED
```

Comment:
I don't see where the default confidence is set now ? Or did we remove the UNDEFINED value in favor of None ?

Reply:
On L116 this functions calls `self.linter.add_message(msgid, line, node, args, confidence, col_offset)` which sets the default confidence here:
https://github.com/PyCQA/pylint/blob/b3336c77378f88eb1428d6ff5b6e78bba1177f40/pylint/message/message_handler_mix_in.py#L229-L246

I thought this was duplicate code.

Reply:
Indeed, looks like `lister.add_message` handles that for us already.